privacy-on-beam/pbeam/aggregations_test.go

```diff
@@ -392,23 +389,21 @@ func TestDropUnspecifiedPartitionsInt(t *testing.T) {
 func TestDropUnspecifiedPartitionsFloat(t *testing.T) {
```

Comment:
I feel like this test should be deleted, we already have test for KV above?

Reply:
This one's for floats... the previous one was for ints.